src/main/resources/default/settings_mc.properties

```diff
@@ -31,7 +31,8 @@ source_y=0.0
 
 # Outputkey Settings
 
-event_param=@source,EventNum
+#event_param=@source,EventNum
+event_param=EventNum
```

Comment:
Dont leave it it commented out, just remove it

Reply:
I just left it there for now, as i wasn't sure what the purpose of it was, as it seemed to be something important. But now that nobody is saying anything about it I assume it is not important, given that it is missing anyways. So I now removed it.